### DISCUSSION TITLE
Rain: need help

### DISCUSSION THREAD

**Discussion by faraujof** [primefaces/community#1969] [PrimeFaces Templates] [answered]
Hello, in August 2023 I purchased the "rain" template.
Now, when I started to test the template, I downloaded the rain-layout-5.0.0.zip file, but I saw that it didn't have any documentation. I  discovered that the template was discontinued, and the documentation can be obtained by running the template itself. But as I'm starting to learn, this has become complicated, is there any way to get the documentation, or provide any instructions on how to run the template?

**Comment by melloware** [comment, marked as answer]
@faraujof yes here is a small zip that has a fixed `pom.xml` and `src/main/conf/`

[rain.zip](https://github.com/primefaces/community/files/15369135/rain.zip)

1. Drop these files into your download Rain directoy.
2. Run `mvn clean jetty:run` to run the showcase at http://localhost:8080/rain
3. Build the whole rain template with `mvn clean package`
